src/Demolite.Http/Demolite.Http/Repository/AbstractHttpRepository.cs

```diff
@@ -24,13 +26,56 @@ protected AbstractHttpRepository()
 	/// Default resilience pipeline builder.
 	/// Can be set null if not intended to be used.
 	/// </summary>
-	protected virtual IResiliencePipelineBuilder? ResiliencePipelineBuilder { get; }
+	protected virtual IResiliencePipelineBuilder? ResiliencePipelineBuilder => null;
+	
+	/// <summary>
+	/// Holds pipelines if any are created.
+	/// </summary>
+	protected Dictionary<RequestType, ResiliencePipeline<IFlurlResponse>> ResiliencePipelines;
+
 
 	/// <summary>
 	///     Setup method used to prepare the Flurl Client
 	/// </summary>
 	protected abstract void SetupClient();
 
+	/// <summary>
+	/// Setup method to create ResiliencePipelines dictionary.
+	/// If a ResiliencePipelineBuilder, it will create the pipelines for the respective request type.
+	/// Returns an empty dictionary if ResiliencePipelineBuilder is null. 
+	/// </summary>
+	protected virtual void SetupPipelines()
```

Comment:
I'm thinking if it might be a good idea to move the entire resiliency into a partial class "AbstractHttpRepository.Resilience.cs". 

How do you feel about this?

Reply:
Good point! Makes it all a little cleaner. I'll get right to it :)